src/Components/AffectedClustersTable/AffectedClustersTable.cy.js

```diff
@@ -523,6 +526,26 @@ describe('non-empty successful affected clusters table', () => {
       checkRowCounts(Math.min(DEFAULT_ROW_COUNT, filterData({}).length));
     });
 
+    it('will reset filters but not pagination and sorting', () => {
+      filterApply({ name: '0' });
+      changePagination(PAGINATION_VALUES[0]);
+      cy.get(TOOLBAR)
+        .find(PAGINATION)
+        .find('button[data-action="next"]')
+        .then(($button) => {
+          cy.wrap($button).click();
+        });
+
+      cy.get('th[data-label="Name"]').find('button').click();
+      cy.get(TOOLBAR).find('button').contains('Reset filters').click();
+      cy.get(TOOLBAR).find(CHIP_GROUP).should('not.exist');
+      checkPaginationSelected('0');
```

Comment:
a nit-pick: since, in `checkPaginationSelected` you pass the argument to `eq`, you should pass a number, not string: https://docs.cypress.io/api/commands/eq#Arguments

```suggestion
      checkPaginationSelected(0);
```